hw1103compaundviewnavigationdrawer/src/main/java/com/tashad16a/gmail/hw1103compaundviewnavigationdrawer/fragment/PageFragment.kt

```diff
@@ -0,0 +1,22 @@
+package com.tashad16a.gmail.hw1103compaundviewnavigationdrawer.fragment
+
+import android.os.Bundle
+import android.support.annotation.NonNull
+import android.support.annotation.Nullable
+import android.support.v4.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.tashad16a.gmail.hw1103compaundviewnavigationdrawer.R
+
+class PageFragment : Fragment() {
+
+    @Nullable
+    override fun onCreateView(
+        @NonNull inflater: LayoutInflater,
+        @Nullable container: ViewGroup?,
```

Comment:
if field marked as '?' no need additionally mark it with @Nullable annotation.
Same for @NotNull field

Reply:
```suggestion
        @Nullable
        override fun onCreateView(
        @NonNull inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.fragment_page, container, false)
    }
```